Dockerfile

```diff
@@ -5,8 +5,8 @@ RUN apt-get update && \
     apt-get install --yes --no-install-recommends apt-transport-https && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
-RUN echo 'deb https://deb.nodesource.com/node_14.x buster main' > /etc/apt/sources.list.d/nodesource.list && \
```

Comment:
Oops, revert this. I was working around our firewall and these changes aren't really needed.